tox.ini

```diff
@@ -7,11 +7,12 @@ deps =
     pytest
     pytest-cov
     jinja2
+    mypy_extensions==0.2.0
```

Comment:
We should just have tox include some shared deps via `-r` here so that we don't need to repeat ourselves a bunch.